### DISCUSSION TITLE
或许可以整合一些其他课堂上常用的小功能

### DISCUSSION THREAD

**Discussion by HurtGhostMeow** [Class-Widgets/Class-Widgets#119] [建议]
或许可以在桌面上实现一个置顶小组件，用来实现**更多的功能**
同时小组件内的**功能模块化**，分离成多个小模块以便维护和更新
1.通知
2.倒计时
3.抽选
4.白板
5.快速启动

**Comment by jiangyin14** [comment]
看看InkCanvasForClass？

**Reply by HurtGhostMeow** [reply]
如果能把多个功能整合在一起就更方便部署了

**Reply by MKStoler4096** [reply]
主要是很容易大而不精

**Reply by HurtGhostMeow** [reply]
> 主要是很容易大而不精

也是
